_docs/style/colors.md

```diff
@@ -10,82 +10,553 @@ status: complete
+
+## The Helix color span
+
+Introductory text and a visual with hex codes...
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Guidelines
+
+Consider the following guidelines when you construct a user interface (UI) that
+includes the Helix color palette:
+
+- xxxx
+- xxxx
+- xxxxx
+- xxxxx
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct a (UI) that
+includes the Helix color palette:
+
+- xxxx
+- xxxx
+- xxxxx
+- xxxxx
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Accessibility
+
+Consider the following accessibility Guidelines when you construct a (UI) that
+includes the Helix color palette:
+
+- xxxx
+- xxxx
+- xxxxx
+- xxxxx
+
+### Color combination Do's and Don'ts
+
+A list of do's and don'ts...
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+## Application frame colors
+
+Description and visual of application frame colors...
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Element with light fill or border"] [class:"image bg-light border"] %}
+
+<img src="{{site.url}}/assets/images/style/colors/xxxxx.png" width="402"/>
+
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Status colors
+
+The Helix color palette includes an array of colors that you can use to communicate the status of an entity, object, or process..
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Criticality of information
+
+Text here
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Element with light fill or border"] [class:"image bg-light border"] %}
+
+<img src="{{site.url}}/assets/images/style/colors/xxxxx.png" width="402"/>
+
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Health status
```

Comment:
Let's talk about what visuals should be. these are fuzzy and they shouldn't include those redline. the redlines are really just reminders of content that needs to go in this section of the docs, not content that should be in the visual.